concrete/controllers/element/navigation/dashboard_menu.php

```diff
@@ -33,11 +29,23 @@ public function displayChildPages(Page $page)
         }
     }
 
+    public function displayTopLevelMenu(Page $page)
```

Comment:
You can drastically shorten this method and make it more readable/maintainable when (if) more paths will be added:

```
    public function displayTopLevelMenu(Page $page)
    {
        $pagePath = $page->getCollectionPath();
        
        return in_array($pagePath, ['/dashboard/welcome', '/dashboard/system']) ? true : $pagePath;
    }
```

Reply:
thanks again ramon I've updated the PR per your suggestion.